live.html

```diff
@@ -299,40 +299,41 @@ <h2 class="font-size-h4 text-gray-light">Ways to Watch</h2>
         <section class="soft-top mobile-hard font-size-smaller font-family-condensed text-center">
           <figure class="wow fadeIn col-sm-2 col-sm-offset-1 mobile-push-half-bottom">
             <div class="push-center w-60">
-              <img class="width-100" src="//crds-cms-uploads.imgix.net/content/images/streaming--watch-desktop.png?auto=format,compress&w=33&h=23&fit=crop" alt="stream Crossroads on desktop" data-optimize-img />
+              <img data-src="//crds-cms-uploads.imgix.net/content/images/streaming--watch-desktop.png" alt="" class="img-responsive imgix-fluid" title="" />

```

Comment:
We will definitely want to add the data-optimize-img to these resources unless you took them out for a reason.

```suggestion
              <img data-src="//crds-cms-uploads.imgix.net/content/images/streaming--watch-desktop.png?auto=format,compress?w=33&h=22&fit=crop" alt="" class="img-responsive imgix-fluid width-100" title="" data-optimize-img />
```

Is there a reason you pulled those out?

You'll need to rinse/repeat for all the images that were changed.